src/FSharp.Data.GraphQL/Execution.fs

```diff
@@ -470,4 +474,49 @@ let internal execute (schema: #ISchema) doc operationName variables root errors
 // we don't need to know possible types at this point
 SchemaMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> SchemaMetaFieldDef
 TypeMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeMetaFieldDef
-TypeNameMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeNameMetaFieldDef
\ No newline at end of file
+TypeNameMetaFieldDef.Execute <- compileField Unchecked.defaultof<TypeDef -> ObjectDef[]> TypeNameMetaFieldDef
+
+
+type Define with
+    /// Single subscription defined inside either object types or interfaces 
+    static member Subscription(name : string, typedef : ObjectDef<'Res>,
+                               subscribe : ResolveFieldContext -> 'Val -> IObservable<NameValueLookup> -> unit,
```

Comment:
I see two problems here:

1. We can define only single publisher / single subscriber relationship. Multiple subscribers are not allowed.
2. All of the methods existing in `Define` type are reflecting native GraphQL type system definitions. There is no such type as subscription. It's artificial construct from our side and there's a risk that it won't fit GraphQL standard in the future. Therefore subscription specific logic should be done totally in the underlying execution engine.